src/client/js/client/libraries.js

```diff
@@ -0,0 +1,126 @@
+/*globals define*/
+/*jshint browser: true*/
+/**
+ * @author kecso / https://github.com/kecso
+ */
+define(['js/Constants'], function (CONSTANTS) {
+    'use strict';
+    function gmeLibraries(logger, state, storage, saveFunction) {
```

Comment:
nodeSetters have a function called storeNode if they are the same I would use the same name.